oml/lightning/callbacks/metric.py

```diff
@@ -72,6 +89,30 @@ def on_validation_batch_end(
             if is_last_expected_batch:
                 self.calc_and_log_metrics(pl_module)
 
+    def _log_images(self, pl_module: pl.LightningDataModule) -> None:
+        if not isinstance(self.metric, EmbeddingMetrics):
+            return
+
+        for metric_name in flatten_dict(self.metric.metrics):  # type: ignore
```

Comment:
I also think we should move the logic about ignoring metrics to the visualisation method. This method can have `metrics to ignore` as an argument

Reply:
Because having this logic here does not allow us to apply our callback to other metrics with visualisation